pkgs/development/web/postman/default.nix

```diff
@@ -1,18 +1,53 @@
-{ stdenv, lib, gnome2, fetchurl, pkgs, xorg, makeWrapper, makeDesktopItem }:
+{ stdenv, lib, fetchurl, pkgs, makeWrapper, makeDesktopItem, wrapGAppsHook
+, atk, at-spi2-atk, alsaLib, cairo, cups, dbus, expat, gdk-pixbuf, glib, gtk3
+, freetype, fontconfig, nss, nspr, pango, udev, libX11, libxcb, libXi
+, libXcursor, libXdamage, libXrandr, libXcomposite, libXext, libXfixes
+, libXrender, libXtst, libXScrnSaver
+}:
+
+with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "postman-${version}";
-  version = "7.0.7";
+  version = "7.5.0";
 
   src = fetchurl {
     url = "https://dl.pstmn.io/download/version/${version}/linux64";
-    sha256 = "47be1b955759520f3a2c7dcdecb85b4c52c38df717da294ba184f46f2058014a";
+    sha256 = "1g1lg3pba1gqkfj7s4s50qyzl0j7i1wk4fsgwmk6rzmy714bwgg0";
     name = "${name}.tar.gz";
   };
 
-  nativeBuildInputs = [ makeWrapper ];
-
-  dontPatchELF = true;
+  libPath = makeLibraryPath [
```

Comment:
```suggestion
  libPath = lib.makeLibraryPath [
```

Reply:
I would prefer this to be in a let binding, above the derivation (don't feel strongly about this), but it would avoid `$libPath` being set during the build

Reply:
When you say build, do you mean the nix build, e.g. buildPhase?
Wasn't sure how it impacts this expression since we're not doing anything there right now.
`buildPhase = ":";`

Reply:
im pretty sure all keys on a derivation get stringified to a variable for all phases. For example, saying `src = <some package>; unpackPhase = "tar xf $src"`, $src isn't populated because it's special, it's populated because it's an attribute on the derviation set.